src/Elastic.CommonSchema.Serilog/LogEventConverter.cs

```diff
@@ -153,53 +283,45 @@ private static Event GetEvent(LogEvent e)
 			var evnt = new Event
 			{
 				Created = e.Timestamp,
-				Category = e.Properties.ContainsKey("ActionCategory")
-					? e.Properties["ActionCategory"].ToString()
+				Category = e.Properties.TryGetValue(SpecialKeys.ActionCategory, out var c) ? c.ToString() : null,
+				Action = e.Properties.TryGetValue(SpecialKeys.ActionName, out var action) ? action.ToString().Replace("\"", "") : null,
+				Id = e.Properties.TryGetValue(SpecialKeys.ActionId, out var actionId)
+					? actionId.ToString().Replace("\"", "")
 					: null,
-				Action = e.Properties.ContainsKey("ActionName")
-					? e.Properties["ActionName"].ToString().Replace("\"", "")
+				Kind = e.Properties.TryGetValue(SpecialKeys.ActionKind, out var actionKind)
+					? actionKind.ToString().Replace("\"", "")
 					: null,
-				Id = e.Properties.ContainsKey("ActionId")
-					? e.Properties["ActionId"].ToString().Replace("\"", "")
+				Severity = e.Properties.TryGetValue(SpecialKeys.ActionSeverity, out var actionSev)
+					? (long?)long.Parse(actionSev.ToString())
 					: null,
-				Kind = e.Properties.ContainsKey("ActionKind")
-					? e.Properties["ActionKind"].ToString().Replace("\"", "")
-					: null,
-				Severity = e.Properties.ContainsKey("ActionSeverity")
-					? long.Parse(e.Properties["ActionSeverity"].ToString())
-					: 0
+				Timezone = TimeZoneInfo.Local.StandardName
 			};
 
-#if NETSTANDARD
-            evnt.Timezone = TimeZoneInfo.Local.StandardName;
-#else
+			//Why does this get overriden in full framework?
+#if FULLFRAMEWORK
```

Comment:
Good question, it was in there as part of an `if` back when it was switching on `DOTNETCORE` - can now be deleted due to line 297 above.

Reply:
Cool cool, we'll delete it at some point :)